lgc/patch/PatchBufferOp.cpp

```diff
@@ -1664,6 +1673,26 @@ Instruction *PatchBufferOp::makeLoop(Value *const loopStart, Value *const loopEn
   return loopCounter;
 }
 
+// =====================================================================================================================
+// Fix incomplete phi incoming value
+void PatchBufferOp::fixIncompletePhi() {
```

Comment:
Please use the plural form:
```suggestion
// Fix incomplete phi incoming values
void PatchBufferOp::fixIncompletePhis() {
```

Reply:
OK. I also change m_incompletePhi to m_incompletePhis.